content/examples/qubole-cluster.Rmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Using RStudio Server Pro inside of Qubole"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+Qubole users can request [beta access to RStudio Server
```

Comment:
I would reword this to remove the mention of beta access and defer that kind of info to the Quoble docs. Up to you if you want to leave it for some other reason though.